app/Actions/Fortify/CreateNewUser.php

```diff
@@ -93,10 +93,18 @@ public function create(array $input): RedirectResponse | User
         $user->emailUpdates()->create();
 
         if (config('other.invite-only') === true) {
-            Invite::where('code', '=', $input['code'])->update([
+            $invite = Invite::where('code', '=', $input['code'])->withoutTrashed()->first();
```

Comment:
Shouldn't need to do `->withoutTrashed()` here. It's only included above because soft deletion isn't considered for `Rule::exists()` validation.

Reply:
Thanks! I wasn't sure why is it used above so made it explicit just in case. Fixed in my next commit.